src/aesnd.rs

```diff
@@ -152,3 +159,9 @@ impl Aesnd {
         } 
     }
 }
+
+struct Align32<T>(pub T);
+
+fn alloc_sound_buffer(buffer: &[u8]) -> Align32<Vec<u8>> {
+    Align32((*buffer).to_vec())        

```

Comment:
nothing here guarantees that the buffer will be 32-byte aligned when allocated. check the source of `gx::Fifo::with_size()` to see how to do it.

or i can fix it later on, up to you.

Reply:
I don't get what you mean? the repr on align32 must make the [u8] it takes aligned and padded to 32 bytes??

Reply:
... what repr? there's no repr attribute. even if there was, it wouldn't do what you need it to.

Reply:
I see what I did! I used #[repr(C, align(32))] orginnaly in align32 but it isnt there anymore